docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,247 @@ sidebar_position: 2
 
 # Quick Start
 
+_Service Binding Operator should be [installed](./installing-service-binding.md) on the cluster prior to running this quick start guide._
+
 In this section we are providing a sample application that one can deploy and just use to play around.
 
-## K8S
+## Kubernetes
+
+### Application
+
+An application is a process running within a container. Examples include a NodeJS Express application, a Ruby Rails application or a Spring Boot application.
+
+Create a NodeJS application:
+
+```yaml
+kubectl apply -f - << EOD
+---
+apiVersion: apps/v1 ############################### GROUP/VERSION
+kind: Deployment    ############################### RESOURCE
+
+metadata:
+  name: nodejs-app  #################### NAME
+  labels:
+    app: nodejs
+    runtime: nodejs
+    runtime-version: 14-ubi7
+
+spec:
+  selector:
+    matchLabels:
+      app: nodejs
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nodejs
+        deploymentconfig: nodejs
+    spec:
+      containers:
+      - name: nodejs
+        image: nodejs:latest
```

Comment:
this image does not exist in docker hub. We should present something that really works.